lib-impl/src/main/java/org/ergoplatform/appkit/impl/NodeAndExplorerDataSourceImpl.java

```diff
@@ -66,7 +65,7 @@ public class NodeAndExplorerDataSourceImpl implements BlockchainDataSource {
      */
     public boolean performCheckBeforeSend = false;
 
-    public NodeAndExplorerDataSourceImpl(ApiClient nodeClient, @Nullable ExplorerApiClient explorerClient) {
+    public NodeAndExplorerDataSourceImpl(ApiClient nodeClient, ExplorerApiClient explorerClient) {
```

Comment:
the explorer is nullable

Reply:
Your reply in discord: "Yes you can remove it. The package is indeed weird normally it is from another namespace."